velox/exec/HashAggregation.h

```diff
@@ -72,6 +72,16 @@ class HashAggregation : public Operator {
 
   RowVectorPtr getDistinctOutput();
 
+  // Setup the projections for accessing grouping keys stored in the grouping
+  // set. 'groupingKeyInputChannels' provides the map from the key column index
+  // in the grouping set (as vector index) to key column channel in the input
+  // (as vector value). 'outputChannelProjections' provides the map from the key
+  // column channel in the output (as vector index)  to the key column index in
+  // the grouping set (as vector value).
```

Comment:
// Creates the projections for accessing grouping keys stored in grouping
// set.
// For 'groupingKeyInputChannels', the index is the key column index from
// the grouping set, and the value is the key column channel from the input.
// For 'outputChannelProjections', the index is the key column channel from
// the output, and the value is the key column index from the grouping set.